BUILDING.md

```diff
@@ -296,25 +296,29 @@ Extract the files and put them in a separate directory (need not be where you ev
 
 GMT can be built on any platform supported by CMake. CMake is a cross-platform,
 open-source system for managing the build process. The building process is
-controlled by two configuration files in the `cmake` directory:
+controlled by three configuration files in the `cmake` directory:
```

Comment:
```suggestion
controlled by two configuration files in the `cmake` directory:
```
Only two configuration files, right?

Reply:
Well, as written up (From Florian), he counts the CMakeDefaults as one of them.  He then says NOT to edit that one.  So that leaves 2 more, for a total or 3.  We could only two for users to make changes to or something.

Reply:
OK with 3.